packages/java/src/node/java-contribution.ts

```diff
@@ -34,12 +38,22 @@ configurations.set('linux', 'config_linux');
 @injectable()
 export class JavaContribution extends BaseLanguageServerContribution {
 
-    @inject(JavaCliContribution)
-    protected readonly cli: JavaCliContribution;
-
+    private javaBundles: string[] | undefined;
     readonly id = JAVA_LANGUAGE_ID;
     readonly name = JAVA_LANGUAGE_NAME;
 
+    constructor(
+        @inject(JavaCliContribution) protected readonly cli: JavaCliContribution,
+        @inject(ContributionProvider) @named(JavaExtensionContribution)
+        protected readonly contributions: ContributionProvider<JavaExtensionContribution>
+    ) {
+        super();
+        this.javaBundles = [];
+        for (const contrib of this.contributions.getContributions()) {
+            this.javaBundles = this.javaBundles.concat(contrib.getExtensionBundles());
```

Comment:
What if multiple bundles contribute the same bundles? Do we need to make sure the list is unique?

Reply:
Even if it will happens, jdt.ls will compare the bundle's information and skip already installed bundle